src/tinyYolov2/TinyYolov2.ts

```diff
@@ -242,7 +243,7 @@ export class TinyYolov2 extends NeuralNetwork<TinyYolov2NetParams> {
   private extractPredictedClass(classesTensor: tf.Tensor4D, pos: { row: number, col: number, anchor: number }) {
     const { row, col, anchor } = pos
```

Comment:
Same here: `const classesData = await classesTensor.array()` and make this function async.